modules/mbedtls/zephyr_init.c

```diff
@@ -37,7 +37,12 @@
 option."
```

Comment:
This could be cleaned up as `CONFIG_MBEDTLS_HEAP_SIZE` has a default value and will be defined whenever `CONFIG_MBEDTLS_ENABLE_HEAP` is enabled.